telegram/_chat.py

```diff
@@ -1167,7 +1167,7 @@ async def set_administrator_custom_title(
         """
         return await self.get_bot().set_chat_administrator_custom_title(
             chat_id=self.id,
-            user_id=user_id,
+            user_id=int(user_id),
```

Comment:
instead of casting to int, we should adapt the type hint of the argument to match the one of `Bot.set_admin_title`. I see that in `test_chat.py`, the corresponding test `test_set_administrator_custom_title` and actually a few other tests are missing the assertions for `check_shortcut_signature` and `check_shortcut_call`. I expect that those would have already caught that …

We should add them. If not done within the scope of this PR. we should open an issue about that.